backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationDataResolver.java

```diff
@@ -20,29 +17,18 @@
  * removed before Christmas 2020.
  */
 @Service
-public class OrganizationDataResolver implements GraphQLResolver<ApiOrganization> {
+public class OrganizationDataResolver implements GraphQLResolver<Organization> {
 
 	@Autowired
 	private OrganizationService _orgService;
 
-	public List<ApiFacility> getTestingFacility(ApiOrganization o) {
-		return o.getTestingFacility();
+	public List<ApiFacility> getTestingFacility(Organization o) {
```

Comment:
Not coincidentally, the next thing that I did after writing this comment was #281 😬 

Because yeah, changing the schema without automated checks is madness.